src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -5252,7 +5252,8 @@ public void EmitSequenceOfBinaryExpressions_03()
             const int step = 4096;
             const int limit = start * 4;
 
-            for (int count = start; count <= limit && diagnostics.IsEmpty; count += step)
+            int count;
```

Comment:
>int count; [](start = 12, length = 10)

It is not clear what motivated this change, it looks like it didn't change behavior of the method.